From b246897a5a8e31b47442d15d7854a0eb8e64e949 Mon Sep 17 00:00:00 2001
From: Li Wang <liwang@redhat.com>
Date: Wed, 14 Jun 2017 11:38:30 +0800
Subject: [PATCH] dio: reduce the number of cycles

These two cases (dio29/30) takes about 100 minutes, but they're not
stress test, so I suggest reducing the number of cycles.

After the change on my RHEL-7.3 (23G RAM, 24 CPU(s), x86_64) box, it
still takes almost 9 mins.

  diotest06    1  TPASS  :  Read with Direct IO, Write without
  diotest06    2  TPASS  :  Write with Direct IO, Read without
  diotest06    3  TPASS  :  Read, Write with Direct IO
  diotest06    0  TINFO  :  3 testblocks 1000 iterations with 100 children completed

  real	8m4.805s
  user	71m36.722s
  sys	8m51.649s

[From Eryu:

 I ran diotest30 10 times with "-i 100" and 8 of them failed. IIRC, it
 was not 100% reproduced either when I first saw the test failure in
 4.10-rc kernel testing. So I think ~80% probability to reproduce is
 fine.]

Signed-off-by: Li Wang <liwang@redhat.com>
Acked-by: Eryu Guan <eguan@redhat.com>
Acked-by: Cyril Hrubis <chrubis@suse.cz>
---
 runtest/dio | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/runtest/dio b/runtest/dio
index a26e001..185cd57 100644
--- a/runtest/dio
+++ b/runtest/dio
@@ -40,8 +40,8 @@ dio27 diotest6 -b 8192 -o 1024000 -i 1000 -v 100
 dio28 diotest6 -b 8192 -o 1024000 -i 1000 -v 200
 
 ### Run the tests with more children
-dio29 diotest3 -b 65536 -n 100 -i 1000 -o 1024000
-dio30 diotest6 -b 65536 -n 100 -i 1000 -o 1024000
+dio29 diotest3 -b 65536 -n 100 -i 100 -o 1024000
+dio30 diotest6 -b 65536 -n 100 -i 100 -o 1024000
 #
 # RAW DEVICE TEST SECTION
 #   DEV1 and DEV2 should be exported prior to execution or
-- 
2.9.3

